hedgehog/src/Hedgehog.hs

```diff
@@ -37,9 +37,9 @@
 --
 -- You can then load the module in GHCi, and run it:
 --
--- >>> tests
--- ━━━ Test.Example ━━━
---   ✓ prop_reverse passed 100 tests.
+-- > λ tests
```

Comment:
It's there already + on my keyboard as well 😉 

![c3-ufqzxuaaymmi](https://cloud.githubusercontent.com/assets/287532/24939924/121874d8-1f49-11e7-88b1-7d459ca5130d.jpg)